frontend/packages/kubevirt-plugin/integration-tests/tests/vm.detail.flavor.scenario.ts

```diff
@@ -36,9 +36,7 @@ describe('KubeVirt VM detail - edit flavor', () => {
       const vm1Config = vmConfig(configName.toLowerCase(), testName, provisionConfig);
       vm1Config.startOnCreation = false;
 
-      const vm = await wizard.createVirtualMachine(
-        vmConfig(configName.toLowerCase(), testName, provisionConfig),
-      );
+      const vm = await wizard.createVirtualMachine(vm1Config);
```

Comment:
Should we use `createResource(testVM)` here? Other tests `vm.detail.*` are using it.

Reply:
I would keep the VM generated in the Wizard for the sake of being sure the VM labels have correct format.